src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -2413,6 +2413,10 @@ export default defineComponent({
       container.value.classList.add('no-cursor')
 
       await performFirstLoad()
+
+      player.addEventListener('ratechange', () => {
+        emit('current-playback-rate-update', video.value.playbackRate)
```

Comment:
As you are listening to the player and not the video element you should also read the value from the player (as you are combining two separate things the values can differ).
```suggestion
        emit('current-playback-rate-update', player.getPlaybackRate())
```